dev/README

```diff
@@ -1,30 +1,30 @@
-This directory contains informations and tools to help developing the
+This directory contains information and tools to help develop the
                              Coq system
                        ======================
 
 
 Debugging and profiling (in current directory - see doc/debugging.txt)
 -----------------------
 
-ocamldebug-coq: to launch ocaml debugger
+ocamldebug-coq: to launch ocaml debugger (generated by the configure script)
 
 db:      to install pretty-printers from ocaml debugger
-base_db: to install raw pretty-printers from ocaml debugger
+base_db: to install raw pretty-printers from ocaml debugger (subset of db--still needed?)
 
-include:      to install pretty-printers from ocaml toplevel
+include:      to install pretty-printers from ocaml toplevel (use with the coq Drop command)
 base_include: to install raw pretty-printers from ocaml toplevel
 
-vm_printers.ml, dev_printers.ml: ML pretty-printers for debugging
+vm_printers.ml, top_printers.ml: ML pretty-printers for debugging
 
 
-Miscellaneous informations about the code (directory doc)
+Miscellaneous information about the code (directory doc)
 -----------------------------------------
 
-changes.txt:   (partial) per-version summary of the evolutions of Coq ML source
+changes.md:   (partial) per-version summary of the evolution of Coq ML source
```

Comment:
Add a space so that this remains indented?

Reply:
There is no "changes.txt" file in the git tree.  I assume "changes.md" replaced it.

Reply:
Yes, it is good that you noticed that we forgot to update this line when `changes.txt` was moved to `changes.md`. I was only talking about the indentation.